notebooks/124-hugging-face-hub/124-hugging-face-hub.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Learn more on <a href="https://huggingface.co/docs/optimum/intel/inference" target="_blank">the Hugging Face website</a>.</p><p>-&gt; Learn more in Optimum Intel documentation</p><p> </p><p>or something similar, it is important that link has clear description which info provided inside, other way it do not give trust to user and will be just skipped</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>fixed</p>